### DISCUSSION TITLE
Tiktok icon

### DISCUSSION THREAD

**Discussion by matekomenda** [lucide-icons/lucide#2368] [Ideas]
Guys!! We do need a tiktok icon!🙏 I gotta use a custom one because the package does not have :(

**Comment by karsa-mistmere** [comment]
Please refer to #1423 and especially #670 

Discussions aren't the place to request new icons, but we aren't accepting new brand icons anyways, since most brand logos – including TikTok's – are protected by copyright and so we're unable to adapt them to match Lucide's style.

Just use https://simpleicons.org/?q=tiktok
